notification-portlet-webapp/src/main/webapp/less/modal.less

```diff
@@ -0,0 +1,94 @@
+.notification-flat {
+  color: #0074b7;
+  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
+  min-height: 36px;
+  box-sizing: border-box;
+  align-items: center;
+  line-height: 1em;
+  font-family: Arimo;
```

Comment:
mmmh, Arimo Font... where is it integrated? :innocent: :stuck_out_tongue_winking_eye: Is it in the webjar ?

Reply:
https://fonts.google.com/specimen/Arimo

It would also be good to include generic font families as fallbacks.
Probably `Arial` and `Sans-Serif`.

Reply:
I honestly copied that section of CSS from our generated CSS from one of our ReactJS projects. I'll just remove that line.